.homeycompose/flow/triggers/action_heater_mode_changed.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": {
-    "en": "Heater changed",
-    "nl": "Verwarming veranderd"
+    "en": "Heater mode changed",
```

Comment:
Could be same "Heater mode has changed" as measure_heater_mode_changed.
In my opinion, this is duplicate trigger for it.

Triggering on the measure_heater_mode_changed is better, as it is based on the value retrieved from CMS.